modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/model/DDMFormLayout.java

```diff
@@ -173,6 +173,22 @@ public void setDefinitionSchemaVersion(String definitionSchemaVersion) {
 	public void setPaginationMode(String paginationMode) {
 		_paginationMode = paginationMode;
 	}
+	
+	public void setPreviousPage(Integer previousPage) {
+		_previousPage = previousPage;
+	} 
+	
+	public void setNextPage(Integer nextPage) {
+		_nextPage = nextPage;
+	} 
+	
+	public Integer getNextPage() {
+		return _nextPage;
+	}
+	
+	public Integer getPreviousPage() {
```

Comment:
@brunofariasdeo, why did you decide to use `Integer` instead of the primitive type `int`? Can you please clarify?

Reply:
@natocesarrego, I've decided to use Integer to make it easier to handle when the form has only one page. This way, nextPage and previousPage variables would be null.

Reply:
Makes sense, @brunofariasdeo. :slightly_smiling_face: I'm not sure what Brian will think about it, but let's move on with this pull and see how it goes. :wink: